_includes/__left-nav.html

```diff
@@ -0,0 +1,30 @@
+<nav {% if site.show_full_navigation %}class="full-navigation"{% endif %}>
+  <ul>
+    <li class="nav-item top-level {% if page.url == '/' %}current{% endif %}">
+      {% assign home = site.html_pages | where: 'url', '/' | first %}
+      <a href="{{ site.baseurl }}/">{{ home.title }}</a>
+    </li>
+  </ul>
+
+  <ul>
+    {% assign grouped = site.process | group_by: 'category' %}
+    {% for group in grouped %}
+      <li class="nav-item top-level {% if group.name == page.category %}current{% endif %}">
+        {% assign items = group.items | sort: 'order' %}
+        <a href="{{ site.baseurl }}{{ items.first.url }}">{{ group.name }}</a>
+        <ul>
+          {% for item in items %}
+            <li class="nav-item {% if item.url == page.url %}current{% endif %}"><a href="{{ site.baseurl }}{{ item.url }}">{{ item.title }}</a></li>
+          {% endfor %}
+        </ul>
+      </li>
+    {% endfor %}
+  </ul>
+
+  <ul>
+    <li class="nav-item top-level {% if page.url == '/changelog/' %}current{% endif %}">
+      {% assign changelog = site.html_pages | where: 'url', '/changelog/' | first %}
+      <a href="{{ site.baseurl }}/changelog/">{{ changelog.title }}</a>
+    </li>
+  </ul>
+</nav>
```

Comment:
After this merge it'd be worth deleting the file and relying on git history to get it back when you need it.